src/rollup/types.d.ts

```diff
@@ -3,6 +3,10 @@ import * as ESTree from 'estree';
 
 export const VERSION: string;
 
+export interface WebAssemblyJSAst {
+	Program: object;
+}
```

Comment:
I'll generate TS typing to have a better integration here

Reply:
Not to worry about this - I think it may be best to keep WASM off this outward user-facing API initially. ModuleJSON is a semi-private caching interface for watched rebuilds.

Reply:
I didn't realized at first that this is intended to be used by users. If I generate the TS typings on my side the user could still use them from NPM.